CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Year dependent failing snapshot test
 - Browser test that would always fail until march
+- While editing own information, there was "duplicate" cancel button. 
```

Comment:
How about

>- Duplicate cancel button in information editing view

Reply:
True, that makes more sense. I was staring this line like 5mins and that was the best I came up with 😅